examples/src/main/python/hbase_inputformat.py

```diff
@@ -72,6 +75,9 @@
         keyConverter=keyConv,
         valueConverter=valueConv,
         conf=conf)
+    # hbase_rdd is a RDD[dict]
```

Comment:
Python people may be not familiar with `RDD[dict]`, how about just `RDD of dict`?